site/src/component/ReviewForm/ReviewForm.tsx

```diff
@@ -358,6 +357,10 @@ const ReviewForm: FC<ReviewFormProps> = (props) => {
                   {overCharLimit ? (<p style={{ color: 'red' }}>Your review exceeds the character limit</p>) : null}
                   <p style={content.length > 500 ? { color: 'red' } : {}} className='chars'>{content.length}/500</p>
                 </div>
+                <Form.Text>
+                  <Icon name='info circle' />
```

Comment:
@aahei can you maybe change this to a warning icon (exclamation mark ! or something that shows it's a warning)?